### ISSUE TITLE
Farmers standing still

### ISSUE THREAD

**Issue by IAmAFrenchFry** [TangoTek/TekTopia-Community#98]
**Describe the bug**
Farmers are standing still not doing anything. There are things to harvest and they have items in their inventory, but they are just standing still. I tried pushing them to different places but still nothing.

**To Reproduce**
1. Watch a farmer's behavior

**Expected behavior**
Farmers harvesting crops, delivering items, going to sleep, and moving around in general.

I was in a singleplayer test world.

**Comment by binksayres** [comment]
Seen this issue a couple of times as well and only with the farmers, also in Single Player.  Reloading the world got them moving again.

**Comment by IAmAFrenchFry** [comment]
As in relogging into the world??

**Comment by IAmAFrenchFry** [comment]
?

**Comment by binksayres** [comment]
Yes.  Saving and closing the world, then reopening it seemed to work  ( I only mention because you stated that you are in a single player world).

**Comment by Geex204** [comment]
This started happening to me since 1.0.21. Everyone now and then the Farmers just stop and won't move for a while no matter how much work is to be done or how late it will get. But at some point at night time they suddenly wake up and move to thier bed.
![2019-06-03_12 29 42](https://user-images.githubusercontent.com/51030223/58795837-fd08ab00-85fb-11e9-8fd4-96c2975f3568.png)

**Comment by JohnyDL** [comment]
This might be related to one of the pathing issues, chests and other things not on the floor in storage and more recently trapped chests in storage... The villagers also currently see each half of a double chest separately so if they target a second half without a space to stand right next to it like the one in the corner here they get stuck doing thier jobs. I've seen randomly workers stop even with those things in mind but it's a first step to a solution
![2019-06-03_12 17 16](https://user-images.githubusercontent.com/5202568/58798081-9a161480-85f9-11e9-9bc4-0a398daddf24.png)
